packages/conf-rust-2021/conf-rust-2021.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "contact@trili.tech"
+authors: [
+  "TriliTech"
+  "Simon Cruanes"
+]
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+extra-files: ["test.rs" "sha256=536e506bb90914c243a12b397b9a998f85ae2cbd9ba02dfd03a9e155ca5ca0f4"]
+build: [
+  ["cargo" "--version"]
+  ["rustc" "--edition" "2021" "test.rs"]
+]
+depexts: [
+  ["cargo"] {os-distribution = "centos" & os-version >= "8"}
+  ["cargo"] {os-distribution = "ol" & os-version >= "8"}
```

Comment:
```suggestion
  ["cargo"] {os-distribution = "ol" & os-version > "8"}
```

although Idk what version of cargo/rust is avialable in what version of oraclelinux so you need to check the specifics.

Reply:
This suggestion is wrong. See thread about alpine.